packages/core/src/storages/request_queue.ts

```diff
@@ -883,3 +971,42 @@ export interface RequestQueueOptions {
      */
     proxyConfiguration?: ProxyConfiguration;
 }
+
+export interface AddRequestsBatchedOptions extends RequestQueueOperationOptions {
+    /**
+     * Whether to wait for all the provided requests to be added, instead of waiting just for the initial batch of up to 1000.
```

Comment:
```suggestion
     * Whether to wait for all the provided requests to be added, instead of waiting just for the initial batch of up to `batchSize`.
```